backend/test/services/schema.spec.ts

```diff
@@ -0,0 +1,52 @@
+import { describe, expect, test, vi } from 'vitest'
+
+import { createSchema, findSchemasByKind } from '../../src/services/v2/schema.js'
+import { testModelSchema } from '../testUtils/testModels.js'
+
+const mockSchema = vi.hoisted(() => {
+  const mockedMethods = {
+    save: vi.fn(),
+    deleteOne: vi.fn(),
+    find: vi.fn(() => ({ sort: vi.fn(() => ['schema-1', 'schema-2']) })),
+  }
+
+  const Schema: any = vi.fn(() => ({
+    save: mockedMethods.save,
+  }))
+  Schema.find = mockedMethods.find
+  Schema.deleteOne = mockedMethods.deleteOne
+
+  return {
+    ...mockedMethods,
+    Schema,
+  }
+})
+vi.mock('../../src/models/v2/Schema.js', () => ({
+  SchemaKind: {
```

Comment:
I think you should import this (via e.g. importActual), instead of mocking it.  Alternatively, we could put it in a separate `types` folder.